File: atividade_nicolas/alunos/nicolas_g1/Kelson/combustivel.js
import { input,print } from "../io_utils.js";

function main(){
    const rendimento_alcool = input('Informe o rendimento do carro com alcool: ')
    const rendimento_gasolina = input('Informe o rendimento do carro com gasolina: ')
    const distancia = input('Digite a distancia percorrida: ')
    const valores_alcool_gasolina = input('Digite os valores da gasolina e alcool: ')
    const [a,b] = valores_alcool_gasolina.split(" ")

    const resultado1 = alcool(rendimento_alcool,distancia,a)
    const resultado2 = gasolina(rendimento_gasolina,distancia,b)

    print(`Valor a ser usado em alcool: ${resultado1}`)
    print(`Valor a ser usado em alcool: ${resultado2}`)
}
function alcool(r1,distancia,a){
    const valor = (distancia/r1)*a
    return valor
}

function gasolina(r2,distancia,b){
    const valor = (distancia/r2)*b
    return valor
}
main()

File: atividade_nicolas/alunos/nicolas_g1/kleberson/combustivel.py
combustivel = int(input('Digite 0 se for alcool ou 1 se for gasolina: '))
distancia_a_ser_percorrida = int(input('Informe a distância a ser percorrida(km): '))

if combustivel == 0:
    rendimento_com_alcool = int(input('Informe o rendimento do carro com álcool(km): '))
    valor_litro_alcool = int(input('Valor do litro do álcool: '))
    valor_a_pagar_alcool = distancia_a_ser_percorrida * valor_litro_alcool / rendimento_com_alcool
    print('O valor a ser pago usando alcool é {} R$'.format(valor_a_pagar_alcool))

if combustivel == 1:
    rendimento_com_gasolina = int(input('Informe o rendimento do carro com gasolina(km): '))
    valor_litro_gasolina = int(input('Valor do litro da gasolina: '))
    valor_a_pagar_gasolina = distancia_a_ser_percorrida * valor_litro_gasolina / rendimento_com_gasolina
    print('O valor a pagar usando gasolina é {} R$'.format(valor_a_pagar_gasolina))

'''
se o carro rende 10/l e o litro for 2,00 e a distância a ser percorrida for 40km
vou pagar = 40 * 2 / 10 == 8,00
'''